org.OpenT2T.Sample.SuperPopular.Thermostat/Wink Thermostat/v2/thingTranslator.js

```diff
@@ -0,0 +1,117 @@
+"use strict";
+
+var q = require('q');
+var wh = require("opent2t-translator-helper-wink");
+
+// This code uses ES2015 syntax that requires at least Node.js v4.
+// For Node.js ES2015 support details, reference http://node.green/
+
+const EventEmitter = require("events");
+
+function validateArgumentType(arg, argName, expectedType) {
```

Comment:
Agreed, I didn't intend for this to be fixed now.

---
In reply to: [73779662](https://github.com/openT2T/translators/pull/37#discussion_r73779662) [](ancestors = 73779662,73768373)